tests/temp/tempFileTestInternalApi.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+require_once(dirname(__FILE__) . "/../../interface/globals.php");
+?>
+<html>
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+
+    <script language="JavaScript">
+        function testApi() {
+            $.ajax({
+                type: 'GET',
+                url: '../../apis/api/facility',
+                dataType: 'json',
+                headers: {
+                    'apicsrftoken': <?php echo js_escape($_SESSION['api_csrf_token']); ?>
+                },
+                success: function(thedata){
+                    var thedataJSON = JSON.stringify(thedata);
+                    $("#api").html(thedataJSON);
+                },
+                error:function(){
+                }
+            });
+        }
+        $(document).ready(function(){
+            testApi();
+        });
+    </script>
+
+
+</head>
+
+<?php
+// Looking into how to use the api locally via php calls (3 methods are shown below)
+
+
+// CALL via ajax Rest
+echo "<div id='api'></div>";
+echo "<br><br>";
+
+// CALL via route handler
```

Comment:
Second use case:
Calling api within php server. I chose this mechanism over the previous guzzle mechanism since it bypasses guzzle (primary issue is that in the days of docker one may not be able to compute how to connect to itself automatically; secondary issue is security issues).

Reply:
Example of this mechanism in the main codebase (of this PR) here:
https://github.com/openemr/openemr/blob/add94fb8be361bf34fdf4ae999ec7a14c616d8d2/phpfhir/libs/controller/ClientAppController.php#L75-L78